include/exadg/solvers_and_preconditioners/multigrid/multigrid_input_parameters.h

```diff
@@ -134,14 +135,21 @@ enum class MultigridSmoother
 std::string
 enum_to_string(MultigridSmoother const enum_type);
 
+enum class AMGType
+{
+  Trilinos,
+  Boomer
```

Comment:
Just as a question before making any changes: Do we think `Boomer` is the best name given that the naming in deal.II mostly distinguishes between `Trilinos` and `PETSc`? For example, we do not use `ML` or `MueLu` naming in the `Trilinos` case?

@kronbichler , @peterrum could you also briefly share your opinion?

Reply:
In my opinion we should acknowledge here that `BoomerAMG` belongs to the `hypre` project which is not part of `PETSc`. This is different with `Trilinos` and `ML`. `PETSc` has its own AMG module `GAMG`.

Reply:
Question: `Trilinos` and `ML` is similar to `hypre` and `BoomerAMG`? Would this imply that `ML` would be more appropriate instead of `Trilinos` (in case we use `Boomer`)?

Reply:
I also prefer to be more explicit. Maybe, we could rename `AMGType::Trilinos` to  `AMGType::ML`.

_Side remark: let's nor abbreviate `BoomerAMG` with `Boomer`. I guess the first is a wordplay: "boomerang" (the second a character from BSG) ;)_

Reply:
I think the names of the wrapper classes in deal.II are a bit confusing, but the software packages do not make it easy. At the present stage, I would like the name `Hypre` best, which is the analogous name to `Trilinos`. The alternative, and possibly more oriented to the method that is used and not the package by which we associate it, is to call the case `Boomer` or `BoomerAMG` here and rename what we call `Trilinos` into `ML` (to open the possibility to use MueLu alongside ML on the Trilinos side at some point). I think that latter option is the terminology I saw most frequently at conferences when people just "used" an AMG, if that helps with what we decide for our names.

Reply:
there were new updates coming in while I replied, it seems we favor the `ML/BoomerAMG` case then?

Reply:
I agree. `BoomerAMG` and `ML` would only be consequent. If all of you agree I can change this in this PR.

Reply:
Sound like a good plan. I think this is consistent with the open-closed principle in the sense that we could add `GAMG` or `MueLu` in the future without making changes to the existing code.